frontend/README.md

```diff
@@ -54,6 +54,21 @@ npm run storybook
 
 This will start the storybook dev server on [http://localhost:6006/](http://localhost:6006/) and will open it up on your default web browser.
 
+## Testing
+
+1. Make sure you are in the frontend directory
+2. Run the test script
+
+```shell
+npm run test
+```
+
+### Testing individual files
+
+```shell
+npm run test -- path/to/component.test.tsx
+```
+
```

Comment:
We would also explain how to run the tests in watch mode which I think is an immensly useful feature.

I can't suggest a code block in with the suggestions here :/

### Testing in watch mode

To execute those tests for which changes to the code are detected during development.
```shell
npm run test -- --watch
```

Reply:
This is a great suggestion. I will add it asap